helm/templates/coder.yaml

```diff
@@ -66,7 +66,7 @@ spec:
             {{- if .Values.coder.workspaceProxy }}
             - wsproxy
             {{- end }}
-            - server
```

Comment:
This needs to be something like:
```
if .Values.coder.commandArgs is set:
  toYaml .Values.coder.commandArgs
else:
  if .Values.coder.workspaceProxy:
    - wsproxy
  - server
```

With the way it is now, it'll always put `wsproxy` in front of the custom args if workspaceProxy is set which we don't want, and it'll never put `server`